From: Didier Roche <didrocks@ubuntu.com>
Date: Wed, 20 Jun 2018 19:22:06 +0200
Subject: Change default background color based on session.

We changed default GDM color for our ubuntu session. Change it as well here,
still applying the background noise loading.
This screen is appearing for a slip second when the Shell is loading.
Forwarded: not-needed
===================================================================
---
 js/ui/background.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/js/ui/background.js b/js/ui/background.js
index b24f825..9b0a4c1 100644
--- a/js/ui/background.js
+++ b/js/ui/background.js
@@ -102,12 +102,13 @@ const Lang = imports.lang;
 const Meta = imports.gi.Meta;
 const Signals = imports.signals;
 
+const Desktop = imports.misc.desktop;
 const LoginManager = imports.misc.loginManager;
 const Main = imports.ui.main;
 const Params = imports.misc.params;
 const Tweener = imports.ui.tweener;
 
-var DEFAULT_BACKGROUND_COLOR = Clutter.Color.from_pixel(0x2e3436ff);
+var DEFAULT_BACKGROUND_COLOR = (Desktop.is("ubuntu")) ? Clutter.Color.from_pixel(0x2c001eff) : Clutter.Color.from_pixel(0x2e3436ff);
 
 const BACKGROUND_SCHEMA = 'org.gnome.desktop.background';
 const PRIMARY_COLOR_KEY = 'primary-color';
